src/sentry/snuba/query_subscription_consumer.py

```diff
@@ -197,4 +199,5 @@ def parse_message_value(self, value):
             metrics.incr("snuba_query_subscriber.message_payload_invalid")
             raise InvalidSchemaError("Message payload does not match schema")
 
+        payload["timestamp"] = parse_date(payload["timestamp"]).replace(tzinfo=pytz.utc)
```

Comment:
I figured it made sense to just start passing dates around since we're changing this